manifests/master.pp

```diff
@@ -138,6 +138,8 @@
 #   list is Java version dependent and you will need to check the system Java
 #   documentaiton for details.
 #
+#   * The default is in the module data
```

Comment:
Line 139 above has a typo: documentaiton -> documentation.  Would you please fix and regenerate the REFERENCE.md?